src/Views/SimulatorLog/SimulatorLog.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import "./SimulatorLog.css";
+import { useSelector } from "react-redux";
+import { getCurrentExecution, IExecutionLogState } from "./simulatorLogSlice";
+
+/**
+ * Component for logging the commands of the simulator.
+ */
+export const SimulatorLog: FunctionComponent = () => {
+  const [tableData, setTableData] = useState({} as IExecutionLogState);
+  const currentExecution = useSelector(getCurrentExecution());
+
+  useEffect(() => {
+    setTableData(currentExecution);
+  }, [currentExecution]);
+
+  return (
+    <table>
+      <thead
+        style={{
```

Comment:
Can you move the inline styles to `SimulatorLog.css`? I don't think you need to do "display" here.

Reply:
moved to css but whenever I remove display the UI looks weird